config/locales/server.en.yml

```diff
@@ -2202,6 +2208,7 @@ en:
     username:
       short: "must be at least %{min} characters"
       long: "must be no more than %{max} characters"
+      too_many_codepoints: "is too long"
```

Comment:
I am not sure the key here is a bit confusing, as a programmer I understand it, as a translator, this is confusing